Controls/Tests/Test_Cafe.c

```diff
@@ -21,13 +21,65 @@ int main(void){
 	OS_CPU_SysTickInit();
 
 	// Create any semaphores/mutexes and initialize any global variables here
+	OSMutexCreate(&RegisterOccupied_Mutex, "Register Mutex", &err);
```

Comment:
What is the difference between a mutex and a semaphore?

Reply:
A mutex is mainly used for shared resources. It will be set to pending when a task is using a certain resource to make sure no other tasks interact with it. It is then released, and other tasks are allowed to use it. Semaphors allow tasks to wait for a signal from another task. It may not necessarily be resource-related. It is generally when a process is waiting for something to happen.